cli/module_generate/scripts/generate_stubs.py

```diff
@@ -3,15 +3,97 @@
 import subprocess
 import sys
 from importlib import import_module
+from typing import List
 
 
-def return_attribute(resource_name: str, attr: str) -> ast.Attribute:
+def get_final_imports(imports: List[str]) -> List[str]:
+    final_imports = []
+    for i in imports:
+        if i not in final_imports:
+            final_imports.append(i)
```

Comment:
You're right; I made this change because the order of the imports did matter. We were importing something by using something like `sys.version`. Since then, however, I did make changes so that we no longer need this import, so we can put this back to how it used to be. Thanks for catching!